UnityProject/Assets/Scripts/Player/PlayerSprites.cs

```diff
@@ -91,6 +91,8 @@ public class PlayerSprites : MonoBehaviour
 
 	public bool RootBodyPartsLoaded = false;
 
+	public GameObject OverlaySprites;
```

Comment:
is any other script going to interact with this field? If not, make it private and add ``SerializeField`` attribute instead of making it public.